pkg/project/project.go

```diff
@@ -30,3 +35,24 @@ func CreateProject(client *occlient.Client, projectName string) error {
 	}
 	return nil
 }
+
+func ListProjects(client *occlient.Client) ([]ProjectInfo, error) {
+	currentProject := client.GetCurrentProjectName()
+	allProjects, err := client.GetProjects()
+	if err != nil {
+		return nil, errors.Wrap(err, "Cannot get all the projects")
+	}
+	projects := []ProjectInfo{}
+	for _, project := range allProjects {
+		var isActive bool = false
```

Comment:
nit: we don't have to explicitly set this to false, `var isActive bool` sets to false anyway, or we could do `isActive := false` if we need to be explicit